bin/log-setup.js

```diff
@@ -2,12 +2,35 @@ var fs = require('fs-extra');
 var path = require('path');
 var log4js = require('log4js');
 
-/**
- * make a log directory, just in case it isn't there.
- */
-module.exports.init = function() {
-    fs.emptyDirSync(path.join(__dirname, '../log'));
+module.exports.init = function(opts) {
+    if (opts.l) {
+        log4js.configure(opts.l);
+    } else {
+        var log4js_opts;
 
-    log4js.configure(path.join(__dirname, '../log4js.json'));
+        if (opts.d) {
+            var logfile = '/var/log/outriggerd.log';
+            if (opts.o) {
```

Comment:
Seems like the check for the -o option should be done whether or not we're in daemonize mode so that `outrigger -o /tmp/outrigger.log` would work even in foreground mode.